spec/models/friendship_spec.rb

```diff
@@ -0,0 +1,8 @@
+require 'rails_helper'
+
+RSpec.describe Friendship do
+  describe "relationships" do
+    it {should belong_to :follower}
+    it {should belong_to :followed}
+  end
```

Comment:
I saw some examples that had this format as well, but I'm still confused as to why it works since `follower` and `followed` are not tables of their own. We can discuss it later, but great job!